user/module/probe_openssl.go

```diff
@@ -492,11 +492,10 @@ func (this *MOpenSSLProbe) saveMasterSecret(secretEvent *event.MasterSecretEvent
 		// check crypto type
 		switch uint16(secretEvent.CipherId & 0x0000FFFF) {
 		case hkdf.TLS_AES_128_GCM_SHA256:
+		case hkdf.TLS_CHACHA20_POLY1305_SHA256:
```

Comment:
use 'case hkdf.TLS_AES_128_GCM_SHA256, hkdf.TLS_CHACHA20_POLY1305_SHA256:`  instead of.....

Reply:
done